node-js-basics-101/node_modules/node-static/lib/node-static/util.js

```diff
@@ -0,0 +1,30 @@
+var fs   = require('fs')
```

Comment:
__[jshint]__
* Misleading line break before ','; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->